duneggd/SubDetector/TMS.py

```diff
@@ -185,10 +201,9 @@ def construct(self, geom):
         double_layer_lv.placements.append(double_pla_4.name)
 
         # Position the thin and thick steel
-        # 91 scint layers but first and last layer is scintillator
-        # so only 92 steel layers, 50 thin and 34 thick and 8 double
+        #80 steel layers, 34 thin, 22 thinck and 24 double.
 
-        n_thin_steel = 50
+        n_thin_steel = 34#50
```

Comment:
Why keep the `#50` comment? What is it supposed to mean? Either describe it better or remove it.